core/amber/src/main/scala/edu/uci/ics/texera/web/service/WorkflowExecutionService.scala

```diff
@@ -34,32 +40,15 @@ class WorkflowExecutionService(
     workflowContext: WorkflowContext,
     resultService: ExecutionResultService,
     request: WorkflowExecuteRequest,
+    errorHandler: Throwable => Unit,
     lastCompletedLogicalPlan: Option[LogicalPlan]
 ) extends SubscriptionManager
     with LazyLogging {
+  val context: WorkflowContext = workflowContext
   logger.info("Creating a new execution.")
+  val executionStateStore = new ExecutionStateStore()
```

Comment:
I don't understand the change to the `errorHandler` and I think it is not correct. Why do both WorkflowExecutionService and WorkflowService have an 'executionStateStore`?